torchci/lib/bot/cliParser.ts

```diff
@@ -49,7 +49,7 @@ mergeOption.add_argument("-f", "--force", {
 });
 mergeOption.add_argument("-l", "--land-checks", {
   action: "store_true",
-  help: "Merge with land time checks. (EXPERIMENTAL! DO NOT USE)",
+  help: "Merge with land time checks. This will create a new branch that is rebased on viable/strict and run additional tests (EXPERIMENTAL)",
```

Comment:
```suggestion
  help: "Merge with land time checks. This will create a new branch with your changes rebased on viable/strict and run additional tests (EXPERIMENTAL)",
```